hack/testing/test-upgrade.sh

```diff
@@ -227,10 +227,11 @@ function verifyUpgrade() {
   [[ -z "$(oc get secret/logging-elasticsearch -o jsonpath='{.data.admin-cert}')" ]] && return 1
 
 ### check that migration was successful
+  PROJ_PREFIX=project.
   if [ $checkMigrate = true ]; then
     for project in $(oc get projects -o 'jsonpath={.items[*].metadata.name}'); do
       [[ "${OPS_PROJECTS[@]}" =~ $project ]] && continue
-      [[ -z "$(oc logs $UPGRADE_POD | grep 'Migration for project '$project': {"acknowledged":true}')" ]] && return 1
+      [[ -z "$(oc logs $UPGRADE_POD | grep 'Migration for project '${PROJ_PREFIX}$project': {"acknowledged":true}')" ]] && return 1
```

Comment:
@ewolinetz https://github.com/richm/origin-aggregated-logging/commit/a8c5361894790aa902214038354aa0502127b147 should handle this case too